src/geometric_algebra/generic/defs.lean

```diff
@@ -0,0 +1,67 @@
+/-
+Copyright (c) 2020 lean-ga Team. All rights reserved.
+Released under MIT license as described in the file LICENRE.
+Authors: Utensil Rong
+-/
+import ring_theory.algebra
+import linear_algebra.quadratic_form
+import tactic
+import tactic.lint
+
+-- import geometric_algebra.generic.operators
+
+universes ur uv ug
+
+variables {R : Type ur} [field R]
+variables {V : Type uv} [add_comm_group V] [vector_space R V]
+variables {G : Type ug} [ring G]
+
+section prio
+-- set_option default_priority 200 -- see Note [default priority]
+set_option default_priority 100
+set_option old_structure_cmd true
+
+/--
+-/
+@[ancestor algebra]
```

Comment:
What does this do that isn't covered by `extends`?

Reply:
Still trying to figure out the exact effect, it shows up in group.defs in mathlib

Reply:
Yeah, I've seen it before too, was hoping you'd worked out its meaning since I haven't yet...